src/Tribe/Tickets.php

```diff
@@ -613,9 +613,12 @@ public function delete_ticket( $post_id, $ticket_id ) {}
 		 * @param int   $post_id
 		 * @param int   $ticket
 		 * @param array $raw_data
-		 * @return mixed
+		 *
+		 * @return int|false The updated/created ticket post ID or false if no ticket ID.
 		 */
-		public function save_ticket( $post_id, $ticket, $raw_data = [] ) {}
+		public function save_ticket( $post_id, $ticket, $raw_data = [] ) {
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 72dfe5d7fd3e1a4ac85a3bfcb456afa6</sub>